EnglishLanguageArts/PartsOfSpeech/parts-of-speech.ipynb

```diff
@@ -208,6 +218,16 @@
     "    title='Cumulative Pronouns in Anne of Green Gables',labels=dict(index='Chapter'))"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "anne.drop(columns=['Chapter Text','Length']).plot(\n",
```

Comment:
Since this is a separate code cell, adding a code comment at the top of the cell to explain what's happening would be good